pennylane/drawer/circuit_drawer.py

```diff
@@ -59,23 +59,24 @@ def __init__(
         charset=None,
         show_all_wires=False,
         max_length=None,
+        _offsets=None,
```

Comment:
A docstring entry is missing for this argument.

Reply:
I was wondering if we exclude it from the public API somehow as it's only used internally but I'll add a docstring :)